qt/scientific_interfaces/test/ISISReflectometry/Runs/RunsPresenterTest.h

```diff
@@ -646,12 +712,41 @@ class RunsPresenterTest : public CxxTest::TestSuite {
     EXPECT_CALL(m_view, getSelectedSearchRows())
         .Times(1)
         .WillOnce(Return(selectedRows));
+    m_searchResult = SearchResult("", "", "");
     for (auto rowIndex : selectedRows)
       EXPECT_CALL(*m_searcher, getSearchResult(rowIndex))
           .Times(1)
           .WillOnce(ReturnRef(m_searchResult));
   }
 
+  // Set up a valid search result with content and return the corresponding
+  // model
+  ReductionJobs
```

Comment:
What is this line doing?

Reply:
It's just the return value - clang-format just saw fit to put it on a separate line!

Reply:
Oh yes, I see that now haha!